src/neuroglancer/widget/segment_set_widget.ts

```diff
@@ -208,7 +208,7 @@ export class SegmentSetWidget extends RefCounted {
       this.disabledItems.set(segmentIDString, itemElement);
     }
   }
-
+// TODO: Item Button
```

Comment:
Please add a sentence on what is to be done here, it's not clear from just the comment